src/timeline/TimelineModel.h

```diff
@@ -292,6 +295,8 @@ class TimelineModel : public QAbstractListModel
         bool isEncrypted() const { return isEncrypted_; }
         crypto::Trust trustlevel() const;
         int roomMemberCount() const;
+        bool isDirect() const { return roomMemberCount() <= 2; } // TODO: handle invites
```

Comment:
Invites are not TimelineModels. You probably want to handle the previews, which just have the is_direct flag set in the room info.

Reply:
Actually, I did put in some code for invites/previews/etc. already, just didn't remove the comment (I think that's the case, anyway)